src/refresh.js

```diff
@@ -0,0 +1,9 @@
+const refresh = () => {
+  const btnRefresh = document.querySelector('.refresh');
+
+  btnRefresh.addEventListener('click', () => {
+    window.location.reload();
+  });
+};
+
+export default refresh;
```

Comment:
- [ ] good job on the API calls 👍 however according to the requirements When the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change.

Reply:
Thanks, Sir for your feedback! I have made the changes.